frontend/packages/metal3-plugin/src/components/dashboard/inventory-card.tsx

```diff
@@ -94,6 +94,7 @@ export const InventoryCard: React.FC<InventoryCardProps> = ({
           pluralTitle="Pods"
           count={podCount}
           error={podError}
+          data-test-id="metal3-plugin-dashboard-pod"
```

Comment:
Is the naming based on some convention used in console? I think current name could easily become duplicate if we have several things related to pods in the dashboard.

how about `metal3-plugin-dashboard-inventory-pods` ?